docs_user/modules/proc_adopting-the-openstack-dashboard.adoc

```diff
@@ -2,17 +2,18 @@
 
 = Adopting the {dashboard_service}
 
+To adopt the {dashboard_first_ref}, you patch an existing `OpenStackControlPlane` custom resource (CR) that has the {dashboard_service} disabled. The patch starts the service with the configuration parameters that are provided by the {rhos_prev_long} environment.
+
 .Prerequisites
 
-* Previous Adoption steps completed. Notably, Memcached and
-{identity_service_first_ref} should be already adopted.
+* You have adopted Memcached and the {identity_service_first_ref}.
```

Comment:
@ashu-011 Is Memcached adopted during the "Deploying back-end services" procedure (step 7)? file:///home/kgilliga/github_repo/klgill_data-plane-adoption/data-plane-adoption/docs_build/adoption-user/index-downstream.html#deploying-backend-services_migrating-databases

If yes, I'll link to that procedure in the prerequisites.